src/types/permissions.ts

```diff
@@ -41,3 +41,9 @@ export interface IScopes {
   };
   error: any | null;
 }
+
+export interface IUnconsent {
+  pending: boolean;
+  error: any | null;
+  data: string[];
+}
```

Comment:
This signature is reused. It may make sense to rename it to something generic and use the new interface to define any new item that follows the same signature

Reply:
It is removed by convention when we remove the states in the `src/app/services/reducers/permissions-reducer.ts`.